File: pset6/cash.py
from cs50 import get_float
from math import ceil

dollars = 0
counter = 0

while dollars <= 0:
    dollars = get_float("Change Owed: ")
    if dollars >= 0.1:
        break

coins = ceil(dollars * 100)

while coins >= 25:
    coins -= 25
    counter += 1
while coins >= 10:
    coins -= 10
    counter += 1
while coins >= 5:
    coins -= 5
    counter += 1
while coins >= 1:
    coins -= 1
    counter += 1
print(f"I owe you: {counter}")

File: pset1/cash.c
#include<stdio.h>
#include<cs50.h>
#include<math.h>

int main(void)
{
    float dollars;
    int count = 0;
    do
    {
        dollars = get_float("Change Owed: ");
    }
    while(dollars <= 0);

    int coins = round(dollars*100);

    while(coins >= 25)
    {
        coins-=25;
        count ++;
    }
    while(coins >= 10)
    {
        coins-=10;
        count ++;
    }
    while(coins >= 5)
    {
        coins%=5;
        count ++;
    }
    while(coins >= 1)
    {
        coins%=1;
        count ++;
    }
    printf("I owe you %i\n", count);
}
